mne_bids/path.py

```diff
@@ -155,7 +175,7 @@ def copy(self):
         """
         return deepcopy(self)
 
-    def get_bids_fname(self, kind=None, bids_root=None, extension=None):
+    def get_bids_fname(self, kind=None, bids_root=None, ext=None):
```

Comment:
I'm a little confused why we need this method **and** `get_fpath` -- but maybe it's simply too hot here in Paris right now!

Reply:
Yep left it in there because was trying to see how to adjust the funcs that used it! Adjusted all to use `fpath()` property now.